tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/cluster/distribution/TwoWayTwoNodeClusterTest.java

```diff
@@ -153,9 +146,8 @@ private void configureBeforeStart(Configuration... serverConfigs) {
       for (Configuration config : serverConfigs) {
          config.setPersistenceEnabled(true);
          config.setMessageCounterEnabled(true);
-         config.setJournalFileSize(20971520);
-         config.setJournalMinFiles(20);
-         config.setJournalCompactPercentage(50);
+         config.setJournalFileSize(20 * 1024);
```

Comment:
Seems a bit odd it was 20MB before, but only 20KB now? Is there any indication why was the test was significantly _increasing_ the default value before, seems odd if its fine now its being made tiny compared to the default?

Reply:
knowing the history I know.. it was just how we would play back then.

This was pushing the time on the test significantly.



I had done similar cleanups before.

Reply:
looking at the history.. there's no reason for this huge file... most of our tests are using 10 or 20K from upper tests... this one had its own configuration and missed the defaults that were changed.. just ancient history.